### PR TITLE
[xtensor-blas] no absolute path

### PR DESCRIPTION
**Pull request by autoantwort**
For #20469.

### CODE REVIEW

**Review by PhoebeHui — COMMENTED** [review]
Thanks for the PR!

File: ports/xtensor-blas/portfile.cmake [hunk 1]
```diff
@@ -22,9 +22,11 @@ vcpkg_configure_cmake(
...
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/${PORT})
+file(REMOVE "${CURRENT_PACKAGES_DIR}/include/xtensor-blas/xblas_config_cling.hpp")
+
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/${PORT})
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug ${CURRENT_PACKAGES_DIR}/lib)
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/include/xflens/cxxblas/netlib)
```
Comment (PhoebeHui) [inline ports/xtensor-blas/portfile.cmake:32]:
```suggestion
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug" "${CURRENT_PACKAGES_DIR}/lib")
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/include/xflens/cxxblas/netlib")
```

File: ports/xtensor-blas/portfile.cmake [hunk 2]
```diff
@@ -22,9 +22,11 @@ vcpkg_configure_cmake(
         -DDOWNLOAD_GBENCHMARK=OFF
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/${PORT})
+file(REMOVE "${CURRENT_PACKAGES_DIR}/include/xtensor-blas/xblas_config_cling.hpp")
```
Comment (PhoebeHui) [inline ports/xtensor-blas/portfile.cmake:27]: I think replacing the abolute path is better than remove them directly.

Comment (autoantwort) [reply ports/xtensor-blas/portfile.cmake:27]:
```c++

#ifndef XBLAS_CONFIG_CLING_HPP
#define XBLAS_CONFIG_CLING_HPP

#pragma cling add_library_path("/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xtensor-blas_x64-osx/lib64")
#pragma cling add_library_path("/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xtensor-blas_x64-osx/lib32")
#pragma cling add_library_path("/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xtensor-blas_x64-osx/lib")

#ifndef XTENSOR_USE_FLENS_BLAS

#define HAVE_CBLAS 1

#pragma cling load("libblas")
#pragma cling load("liblapack")

#endif

#endif
```
Afaik this is only for the cling c++ interpreter and the folders `lib*` are empty.

Comment (PhoebeHui) [reply ports/xtensor-blas/portfile.cmake:26]: I see.

**Review by PhoebeHui — APPROVED** [review]

**Pull request merged**